finetune_adapter.py

```diff
@@ -34,24 +35,16 @@
 ds_config = {
     "train_micro_batch_size_per_gpu": micro_batch_size,
     "gradient_accumulation_steps": gradient_accumulation_steps,
-    "zero_optimization": {
-        "stage": 2,
-        # "overlap_comm": True,
-        # "allgather_partitions":True,
-        # "allgather_bucket_size": 2e8,
-        # "reduce_scatter":True,
-        # "contiguous_gradients": True,
-        # "round_robin_gradients": True,
-        "offload_optimizer": {"device": "cpu", "pin_memory": True},
-        "offload_param": {"device": "cpu"},
-    },
+    "zero_optimization": {"stage": 2},
 }
 
 
 def main(
-    data_dir: Path = Path("data/alpaca"), checkpoint_dir: Path = Path("checkpoints/stabilityai/stablelm-base-alpha-3b")
+    data_dir: Path = Path("data/alpaca"),
+    pretrained_dir: Path = Path("checkpoints/stabilityai/stablelm-base-alpha-3b"),
+    out_dir: Path = Path("out/adapter/alpaca"),
```

Comment:
In #31, for some reason last minute commits would merge these two directories. Why? I'm pretty sure we want to separate the input directory and output directory.